interface/patient_file/problem_encounter.php

```diff
@@ -261,98 +261,104 @@ function doclick(pfx, id) {
 </script>
```

Comment:
From line 114 to 115, change these lines to this:
`
trcolors['U'] = new Array('var(--black)', 'var(--gray200)'); // unselected
trcolors['K'] = new Array('var(--black)', 'var(--yellow)'); // selected key
`
Delete lines 116 and 117 as they were commented out and replace line 118 with this:
`trcolors['V'] = new Array('var(--black)', 'var(--indigo)'); // selected value
`